doc/book/src/user/metrics.md

```diff
@@ -130,3 +136,7 @@ docker run --detach -p 9090:9090 --volume prometheus-storage:/prometheus --volum
 docker run --detach -p 3030:3030 --env GF_SERVER_HTTP_PORT=3030 --volume grafana-storage:/var/lib/grafana grafana/grafana
 ```
 
+[^1]: This requires a running Docker daemon. See [the relevant section of the
+ Docker Engine manual](https://docs.docker.com/config/daemon/start/). Also note
+that `sudo` is unnecessary if running on macOS.
```

Comment:
I’m a bit confused as to whether/where `sudo` is needed now. You removed it from some of the commands above (including the one that this footnote is linked from, but not the one that calls inspect). This footnote makes it sound like the opposite.

I think one of the two should change
1. leave `sudo` off the commands (except where it’s _always_ needed, if anywhere) and change this footnote to indicate that `sudo` needs to be added on Linux or
2. leave this footnote as is, but change the commands that need `sudo` on Linux to include it.

I lean toward the first one, because better to have a command fail than `sudo` something unnecessarily, IMO, but the second one will work anywhere, so it’s more friendly.

Reply:
`sudo` is never needed (and will actually cause errors) on `macOS` and is always required for every command on linux.